src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -85,8 +87,22 @@ public static partial class FrameworkInfo
         /// <returns>Enumeration of installations</returns>
         public static IEnumerable<FrameworkInstallation> GetInstallations()
         {
-            using var ndpKey = RegistryKey.OpenRemoteBaseKey(RegistryHive.LocalMachine, string.Empty)
-                .OpenSubKey(@"SOFTWARE\Microsoft\NET Framework Setup\NDP\");
+            try
+            {
+                using var ndpKey = Registry.LocalMachine.OpenSubKey(NetFxNdpRegistryKey, false);
+                foreach (var installation in GetInstallationsFromRegistryKey(ndpKey))
+                {
+                    yield return installation;
+                }
+            }
+            catch
+            {
+                return null;
```

Comment:
Build error:

```
/home/runner/work/sentry-dotnet/sentry-dotnet/src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs(100,17): error CS1622: Cannot return a value from an iterator. Use the yield return statement to return a value, or yield break to end the iteration. [/home/runner/work/sentry-dotnet/sentry-dotnet/src/Sentry/Sentry.csproj]
```

Probably best to just return an empty enumerator?

```suggestion
                yield break;
```

Reply:
Actually, you can't use yield inside of a try-catch block

Reply:
Please resolve conversations once it's sorted out